gtda/utils/validation.py

```diff
@@ -154,3 +156,151 @@ def validate_params(parameters, references, exclude=None):
     parameters_ = {key: value for key, value in parameters.items()
                    if key not in exclude_}
     return _validate_params(parameters_, references)
+
+
+def check_list_of_images(X, **kwargs):
+    """Check a list of arrays representing images, by integrating
+    through the input one by one. To pass a test when `kwargs` is ``None``,
```

Comment:
"`kwargs` is empty"?

Reply:
(And same for other validation functions in this PR.)

Reply:
Should be ok now